client/state/data-layer/wpcom/sites/blog-stickers/index.js

```diff
@@ -24,7 +24,7 @@ export function requestBlogStickerList( { dispatch }, action ) {
 			apiVersion: '1.1',
 			onSuccess: action,
 			onFailure: action,
-		} ),
+		} )
```

Comment:
Perhaps Sublime is using the wrong `prettier`? Odd, as I'd pointed it directly at `.nvm/versions/node/v6.10.3/bin/prettier`.